assets/component-facets.css

```diff
@@ -931,20 +931,6 @@ input.mobile-facets__checkbox {
   visibility: hidden;
 }
 
-.product-count .loading-overlay__spinner,
-.product-count-vertical .loading-overlay__spinner {
-  display: none;
-  position: absolute;
-  right: 0;
-  top: 50%;
-  transform: translateY(-50%);
-  width: 1.8rem;
-}
-
-.product-count__text.loading + .loading-overlay__spinner {
-  display: block;
-}
-
```

Comment:
@andrewetchen referring to you [comment](https://github.com/Shopify/dawn/pull/2996#pullrequestreview-1640715481) I removed it in my next [commit](https://github.com/Shopify/dawn/pull/2996/commits/62f2ff8fcef79503b2d34297fd9caba777e04502)